nntrainer/utils/util_simd.h

```diff
@@ -60,11 +60,11 @@ void compute_rotary_embedding_value(unsigned int dim, unsigned int half_,
  * @brief swish function : X = (Y / (1 + exp( -Y ))) * Z
  *
  * @param N number of elements in X
- * @param X __fp16 * for Vector X
- * @param Y __fp16 * for Vector Y
- * @param Z __fp16 * for Vector Z
+ * @param X _FP16 * for Vector X
+ * @param Y _FP16 * for Vector Y
+ * @param Z _FP16 * for Vector Z
  */
-void swish(const unsigned int N, __fp16 *X, __fp16 *Y, __fp16 *Z);
+void swish(const unsigned int N, _FP16 *X, _FP16 *Y, _FP16 *Z);
```

Comment:
@skykongkong8 , I'm not sure How the previous version of this code section compiled properly for the Debian build. Maybe the default of enable_fp16 is false for Debian. 
We need unittest about the swish for each data type ( fp32 and fp16 ) in the next pr.

Reply:
I already conducted unittest on my local, but the reason why it is not here is because I moved this function to `util_simd.h` file from `tensor.h`.
I messed up with this typo issue while moving it.